ext/mysqlnd/mysqlnd_libmysql_compat.h

```diff
@@ -95,7 +95,6 @@
 #define mysql_stmt_send_long_data(s,p,d,l) mysqlnd_stmt_send_long_data((s), (p), (d), (l))
 #define mysql_stmt_attr_get(s,a,v)		mysqlnd_stmt_attr_get((s), (a), (v))
 #define mysql_stmt_attr_set(s,a,v)		mysqlnd_stmt_attr_set((s), (a), (v))
-#define mysql_stmt_param_metadata(s)	mysqlnd_stmt_param_metadata((s))
```

Comment:
This is part of the MySQL API, but given https://dev.mysql.com/doc/c-api/8.0/en/mysql-stmt-param-metadata.html I'd say it's fine to drop it.